devicetypes/zooz/zooz-remote-switch-zen34.src/zooz-remote-switch-zen34.groovy

```diff
@@ -0,0 +1,321 @@
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+*/
+
+import groovy.transform.Field
+
+@Field static Map commandClassVersions = [
+	0x20: 1,	// Basic
+	0x26: 3,	// Switch Multilevel (4)
+	0x55: 1,	// Transport Service
+	0x59: 1,	// AssociationGrpInfo
+	0x5A: 1,	// DeviceResetLocally
+	0x5B: 1,	// CentralScene (3)
+	0x5E: 2,	// ZwaveplusInfo
+	0x6C: 1,	// Supervision
+	0x70: 1,	// Configuration
+	0x72: 2,	// ManufacturerSpecific
+	0x73: 1,	// Powerlevel
+	0x7A: 2,	// Firmware Update Md (3)
+	0x80: 1,	// Battery
+	0x84: 2,	// WakeUp
+	0x85: 2,	// Association
+	0x86: 1,	// Version (2)
+	0x87: 1,	// Indicator
+	0x8E: 2,	// MultiChannelAssociation (3)
+	0x9F: 1		// Security 2
+]
+
+@Field static List<String> supportedButtonValues = ["down","down_hold","down_2x","down_3x","down_4x","down_5x","up","up_hold","up_2x","up_3x","up_4x","up_5x","down_released","up_released"]
+
+@Field static Map configParams = [
+	ledMode: [num:1, title:"LED Indicator Mode", size:1, defaultVal:1, options:[0:"Always off", 1:"On when pressed [DEFAULT]", 2:"Always on (upper paddle color)", 3:"Always on (lower paddle color)"]],
+	upperPaddleLedColor: [num:2, title:"Upper Paddled LED Indicator Color", size:1, defaultVal:1, options:[0:"White", 1:"Blue [DEFAULT]", 2:"Green", 3:"Red", 4:"Magenta", 5:"Yellow", 6:"Cyan"]],
+	lowerPaddleLedColor: [num:3, title:"Lower Paddle LED Indicator Color", size:1, defaultVal:0, options:[0:"White [DEFAULT]", 1:"Blue", 2:"Green", 3:"Red", 4:"Magenta", 5:"Yellow", 6:"Cyan"]]
+]
+
+@Field static int wakeUpInterval = 43200
+@Field static int btnPushed = 0
+@Field static int btnReleased = 1
+@Field static int btnHeld = 2
+@Field static int btnPushed2x = 3
+@Field static int btnPushed6x = 7
+
+metadata {
+	definition (
+		name:"Zooz Remote Switch ZEN34", 
+		namespace:"Zooz", 
+		author: "Kevin LaFramboise (krlaframboise)", 
+		ocfDeviceType: "x.com.st.d.remotecontroller",
+		mnmn: "SmartThingsCommunity",
+		vid: "540fce12-499a-3b90-b276-f4159eb55f42"
+	) {
+		capability "Sensor"
+		capability "Battery"
+		capability "Button"
+		capability "Refresh"
+		capability "Configuration"
+		capability "Health Check"
+		capability "platemusic11009.firmware"
+		capability "platemusic11009.syncStatus"
+
+		//zw:Ss2a type:1800 mfr:027A prod:7000 model:F001 ver:1.01 zwv:7.13 lib:03 cc:5E,55,9F,6C sec:86,85,8E,59,72,5A,73,80,5B,70,84,7A
+		fingerprint mfr: "027A", prod: "7000", model: "F001", deviceJoinName: "Zooz Remote" //Zooz Remote Switch ZEN34
```

Comment:
Multiple sections of that handler would need to be re-written/modified because it only supports 1 scene and the device supports 2.  

The other built-in button DTHs that support multiple scenes use child devices which is a messy approach for this device because it's a paddle switch and the button capability already supports a bunch of "up" and "down" button values.